packages/default-theme/cms/elements/CmsElementCategoryNavigation.vue

```diff
@@ -107,7 +110,7 @@ export default {
   },
   methods: {
     getCategoryUrl(route) {
-      return route.path || ''
```

Comment:
you can use `  { getCategoryUrl } from '@shopware-pwa/helpers'` like it was in packages/default-theme/components/SwTopNavigation.vue

Reply:
👌 good point, fixed